### DISCUSSION TITLE
GOLive

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#265] [General]
Originally posted 2016-01-21 21:23:56.319000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>Î have uploaded a new version of GO Live.

GOLive is a preconfigured operarting system to drive a GO only PC.
Only MIDI assignements are missing. It includes the GO demo set - other samplesets can be downloaded.

* It supports booting from a CD/DVD.
* It supports running from a USB stick or a USB harddisk without any installation.
* It includes a Live installer to permanently install it.

**Comment by larspalo** [comment]
Originally posted 2016-01-28 18:23:24.840000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>I have uploaded a new version of GOLive.

The most user noticeable difference is, that the logout/shutdown functions are gone. Implementing functions in GrandOrgue to shutdown the operation system has never been planed, as avoiding the OS shutdown was the better alternative.

GrandOrgue has already been designed to be &quot;crash safe&quot;. GOLive extends this to the operating system - like it has been in the old MS DOS days. 

The power button of the PC simply turns the PC power off. There is also a menu entry for that. Both option signal power off to the firmware of your computer.

There is also an automatic restart of all relevant tool (including GrandOrgue), if they stop running. Even for really big sampleset, it just takes a few seconds to restore a running GrandOrgue with the last saved settings.

**Comment by larspalo** [comment]
Originally posted 2016-05-15 07:28:16.625000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>I pusblished a new GO Live maintenace release.It included the persistent device name support by default.

The persistent device name support is also part of the lastest GrandOrgue source code. The OpenSuSE Tumbleweed, OpenSuSE Factory and Fedora 24 builds include the necessary supports - its enabled, if you use it with a linux kernel &gt;= 4.6 rc1.
For OpenSuSE 42.1, there are patched packages available: http://download.opensuse.org/repositories/home:/e9925248:/branches:/openSUSE:/Leap:/42.1:/Update/standard/

The OS X build system has been updated. An untested builds for OS X 10.11 is availabe in the Testing folder of our download area - feedback is welcome.

**Comment by larspalo** [comment]
Originally posted 2019-07-03 17:42:59.883000 by sussexbob in GrandOrgue discussion forum on SourceForge<br><br>I downloaded the GOLive image and it&#039;s great, works fine, but I can&#039;t find the installer. The only options in the boot menu are Go2Go, Go2GO safe mode, boot from HDD and memory test. I&#039;m sure I must be missing something here...  How do I get this onto my PC as it&#039;s main (and only)  OS please????  Many Thanks.

**Comment by larspalo** [comment]
Originally posted 2019-07-03 22:43:11.533000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>https://sourceforge.net/p/ourorgan/discussion/962125/thread/0cce2b06f5/

**Comment by William-Purvis** [comment]
I'm trying to find something about GOlive, sounds interesting, but can't get into the sourceforge website without signing my life away.
Anyone have any up-to-date info? I decided to go with GrandOrgue about 5 years ago, and am finally getting near to having a
console ready for playing. Tryingto decide what flavour of Linux to use....

**Reply by larspalo** [reply]
Unfortunately, there's no recent GOlive version that I can refer you to. GrandOrgue will work more or less on any recent Linux distribution that support its building. Here on Github we have ready to use builds, https://github.com/GrandOrgue/grandorgue/releases, that will work for many different distributions. When it comes to Linux, you're free! Free to use whatever you like to. Best advice is to test a few different distributions and go with what appeals to you.
